exo-go/features/template/add_template/add_template.feature

```diff
@@ -13,9 +13,15 @@ Feature: adding remote templates
 
 
   Scenario: adding a new service template
-    When running "exo template add boilr-spark https://github.com/tmrts/boilr-spark.git" in my application directory
-    Then my application contains the directory ".exosphere/boilr-spark"
-    And my git repository has a submodule ".exosphere/boilr-spark" with remote "https://github.com/tmrts/boilr-spark.git"
+    When running "exo template add boilr-license https://github.com/tmrts/boilr-license.git" in my application directory
+    Then my application contains the directory ".exosphere/boilr-license"
+    And my git repository has a submodule ".exosphere/boilr-license" with remote "https://github.com/tmrts/boilr-license.git"
+    And my git repository has a submodule ".exosphere/boilr-license" at commit "afb2fa6"
```

Comment:
This step will fail when there are new commits in the `tmrts/boilr-license` repo, correct? I guess that's unlikely but it would be nice if we could make this step check for the latest commit instead.